main/ui/src/main/java/org/cryptomator/ui/ExitUtil.java

```diff
@@ -75,6 +70,23 @@ private void initMinimizeExitHandler(Runnable exitCommand) {
 	private void initTrayIconExitHandler(Runnable exitCommand) {
 		final TrayIcon trayIcon = createTrayIcon(exitCommand);
 		try {
+			// I'm sorry but no lambda here as several methods must be implemented
+			trayIcon.addMouseListener(new MouseListener() {
```

Comment:
Can you use `MouseAdapter` instead? Still no lambdas, of course, but at least only one method to be overwritten.